frontend/src/app/modules/enterprise/services/popup.service.ts

```
@@ -0,0 +1,15 @@
+import {Injectable} from '@angular/core';
+
+@Injectable()
+export class PopupService {
+
+  constructor() {
+  }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)